### ISSUE TITLE
Move from native pipe to tidyverse pipe?

### ISSUE THREAD

**Issue by athowes** [epinowcast/epidist#336]
> native pipe is a more stringent requirement on R version - is it necessary and / or already driven by another dependency?

_Originally posted by @pearsonca in https://github.com/epinowcast/epidist/pull/329#discussion_r1762779645_

**Comment by seabbs** [comment]
I think its fine for us to use the native pipe and just accept a relatively new R version as this is such a new R package

**Comment by athowes** [comment]
Yes I don't know the proportion of people running an old version of R who are in our target audience. I imagine it's small

**Comment by pearsonca** [comment]
seems reasonable to me - wanted to prompt a conscious decision mostly.

However, I do recall that most (all?) of the piping was one stage. If that's correct view, seems like there's a "third way" resolution to this problem, which also happens to match the tidyverse piping guidance (i.e.: don't).

**Comment by athowes** [comment]
https://github.com/search?q=repo%3Aepinowcast%2Fepidist%20%7C%3E&type=code

I think a minority of these are one line, and they don't seem to egregious use of  `|>` to me.

For example arguably something like

```r
data.frame() |>
mutate()
```

is one line but I think it's fine.

**Comment by athowes** [comment]
Based on F2F with @seabbs we import `cmdstanr` which has:

```
Depends:
    R (>= 3.5.0)
```

So aiming to support low R versions a bit hopeless. This plus speed benefits of `|>` we thought was enough to stick with as is for now.
